cr/state/committeerollback_test.go

```diff
@@ -118,6 +267,14 @@ func committeeKeyFrameEqual(first *CommitteeKeyFrame, second *CommitteeKeyFrame)
 		proposalKeyFrameEqual(first.ProposalKeyFrame, second.ProposalKeyFrame)
 }
 
+func checkResult(t *testing.T, A *CommitteeKeyFrame, B *CommitteeKeyFrame,
```

Comment:
Reduce duplication like this:
A, B, C, D *CommitteeKeyFrame

Reply:
change it next time